libs/community/langchain_community/chains/pebblo_retrieval/base.py

```diff
@@ -331,6 +333,10 @@ def _get_app_details(  # type: ignore
             framework=framework,
             chains=chains,
             plugin_version=PLUGIN_VERSION,
+            client_version=Framework(
+                name="langchain_community_version",
```

Comment:
`name` should be just pkg name, i.e., `langchain_community`. Also, is there a way to autodetect the current pkg name and version using utils like `importlib.metadata`?